tools/IceRpc.ProtocGen/ClientGenerator.cs

```diff
@@ -84,9 +84,8 @@ internal static string GenerateImplementation(ServiceDescriptor service)
 /// <remarks>protoc-gen-icerpc-csharp generated this record struct from Protobuf service <c>{service.Name}</c>.</remarks>
 public readonly partial record struct {clientImplementationName} : I{service.Name.ToPascalCase()}
 {{
-    /// <summary>Gets the default service address for services that implement Protobuf service {service.FullName}.
-    /// Its protocol is <c>icerpc</c> and its path is the Protobuf service name.
-    /// </summary>
+    /// <summary>Gets the default service address for services that implement Protobuf service {service.FullName}:
+    /// <c>icerpc:/{service.Name}</c>.</summary>
```

Comment:
```suggestion
    /// <c>icerpc://{service.Name}</c>.</summary>
```
?

Reply:
No: `//` is for the authority. Here, we specify the path.